src/main/groovy/com/epam/petclinic/plugin/QualityAwareJavaPlugin.groovy

```diff
@@ -35,44 +35,45 @@ public class QualityAwareJavaPlugin implements Plugin<Project> {
 
     @Override
     void apply(Project project) {
-        project.plugins.apply(JAVA_PLUGIN_ID)
-
         project.extensions.create(QualityAwareJavaExtension.NAME, QualityAwareJavaExtension)
+        project.childProjects.values().each { subProject ->
```

Comment:
We have task `checkCodeQualityErrors` for all subprojects it gathers errors. I think that is for such purposes if we have to combine all subprojects and run the task from the root.